src/RequestExecutor/HttpRequestExecutor.ts

```diff
@@ -86,11 +86,11 @@ export class HttpRequestExecutor implements RequestExecutor {
       const errorCode = err.cause?.code ?? err.error?.syscall ?? err.name;
 
       logger.error(
-        'Error executing request: "%s %s HTTP/1.1"',
+        'Error executing request: "%s %s HTTP/1.1" \nCause: %s',
```

Comment:
Please revert these changes, it makes it harder to read

Reply:
done